opslevel/datasource_opslevel_rubric_level.go

```diff
@@ -84,13 +84,13 @@ func (d *LevelDataSource) Read(ctx context.Context, req datasource.ReadRequest,
 		return
 	}
 
-	levels, err := d.client.ListLevels()
+	levels, err := d.client.ListLevels(nil)
```

Comment:
This is because of a change in opslevel-go to unify connection types on "list" calls.  It was caught by the golang compiler as I suspected.